src/tags/control/List.js

```diff
@@ -33,22 +33,25 @@ const ListItemModel = types
   }));
 
 /**
- * List element, used for ranking results. Great choice for recomendation systems.
+ * Use the List tag to rank results, for example for recommendation systems.
+ *
+ * Use with the following data types: audio, image, HTML, paragraphs, text
  * @example
+ * <!--Labeling configuration for a list of possible reply options that can be ranked-->
  * <View>
  *  <HyperText name="page" value="$markup"></HyperText>
  *  <List name="ranker" value="$replies" elementValue="$text" elementTag="Text" ranked="true" sortedHighlightColor="#fcfff5"></List>
  * </View>
  * @name List
- * @meta_title List Tags for Lists
- * @meta_description Label Studio List Tags customize Label Studio with lists for machine learning and data science projects.
+ * @meta_title List Tag for Lists
+ * @meta_description Customize Label Studio with lists for machine learning and data science projects.
  * @param {string} elementValue                - Lookup key for a child object
- * @param {Text|Image|Audio} [elementTag=Text] - Element used to render children
+ * @param {Text|Image|Audio} [elementTag=Text] - Object tag used to render children
  * @param {string} value                       - List values
  * @param {string} name                        - Name of group
```

Comment:
I'd like to add more details to these params too but I don't understand them. Can you help?